pkg/trace/api/api.go

```diff
@@ -762,3 +763,34 @@ func getMediaType(req *http.Request) string {
 	}
 	return mt
 }
+
+// getDirector returns a director for a httputil.ReverseProxy. It enriches requests with headers setting custom
+// container and additional tags and overwrites the user agent header.
+func getHeaderDecoratorDirector(tags string, metricSuffix string) func(req *http.Request) {
+	return func(req *http.Request) {
+		req.Header.Set("Via", fmt.Sprintf("trace-agent %s", info.Version))
+		if _, ok := req.Header["User-Agent"]; !ok {
+			// explicitly disable User-Agent so it's not set to the default value
+			// that net/http gives it: Go-http-client/1.1
+			// See https://codereview.appspot.com/7532043
+			req.Header.Set("User-Agent", "")
+		}
+		containerID := req.Header.Get(headerContainerID)
+		if ctags := getContainerTags(containerID); ctags != "" {
+			req.Header.Set("X-Datadog-Container-Tags", ctags)
+		}
+		req.Header.Set("X-Datadog-Additional-Tags", tags)
+		_ = metrics.Count("datadog.trace_agent."+metricSuffix, 1, nil, 1)
```

Comment:
```suggestion
		metrics.Count("datadog.trace_agent."+metricSuffix, 1, nil, 1)
```
It's fine to ignore the error. No need to assign to `_`. If you did that because of your editor, perhaps there is some setting somewhere.